protocol_rfcs/index.md

```diff
@@ -0,0 +1,5 @@
+## Table of RFCs
+
+Format: <RFC file name>, proposed <date>, accepted/rejected <date> 
```

Comment:
Suggestion: We should differentiate proposed vs. accepted vs. stabilized/rejected dates, no?

As in, somebody proposes feature X; after discussion, the proposal is accepted (= can now start iterating on the implementation and spec changes), or rejected (= thanks but no thanks). 

If a deal-breaker is encountered during the development process, then the RFC would transition from "accepted" to "rejected" (or, if we wanted finer grained tracking, "abandoned").

Eventually, somebody proposes to _stabilize_ X. It's either stabilized, or left as "accepted" for the time being, or could be rejected/abandoned (tho the latter outcome would arguably be a community process failure, to let it get so far only to be rejected at the end like that).

Reply:
Update: I just realized that "proposed" here probably means "RFC PR merged" while "accepted" means "RFC formally accepted and PROTOCOL.md updated" ?

That would definitely simplify things!

Reply:
Also, would it make sense to have an actual table, so the above becomes actual column headers?
```suggestion
|Link to RFC | RFC title | Date proposed | Date accepted | Date rejected |
|-|-|-|-|-|
|...|||||
```
... which should render as:

|Link to RFC | RFC title | Date proposed | Date accepted | Date rejected |
|-|-|-|-|-|
|...|||||

Reply:
Finally, should we consider having a different table for each state, so it's easier to tell what's going on?

Imagine a feature that spends a long time in development, so it's still in "proposed" state but far down the list below several other features that already "accepted" and/or "rejected." It would be pretty easy to move an entry's row to a different section, and the result might look like this:

# Proposed

|Link to RFC | RFC title | Date proposed |
|-|-|-|
|...|||

# Accepted

|Link to RFC | RFC title | Date proposed | Date accepted |
|-|-|-|-|
|...||||

# Rejected

|Link to RFC | RFC title | Date proposed | Date rejected |
|-|-|-|-|
|...||||

Reply:
We might also want to link the corresponding github issue as an additional column in the table, since that's where all the discussion is supposed to live?

Reply:
And yes, proposed means RFC merged as the RFC merging and index updating will be done together.